docs/tools/appium/setups/webdriverio/setup-webdriverio-basic.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Basic Setup on TestObject using WebdriverIO
+layout: en
+permalink: docs/tools/appium/setups/webdriverio/basic/
+---
+
+<a href="http://webdriver.io" target="_blank">WebdriverIO</a> is a powerful framework for webtesting. Webtests written using WebdriverIO can easily be configured to run on real mobile devices on TestObject. You can find a working example for running WebdriverIO tests on our github page: <a href="https://github.com/testobject/webdriverio-example" target="_blank">TestObject WebdriverIO example</a>.
```

Comment:
Again, wouldn't make "webtest" a single word, but you are the native speaker so it's your call ;)